docs/advanced/flash.md

```diff
@@ -118,3 +122,13 @@ They test all flash API functionality. To run the tests, use these commands:
 
 - FlashIAP: `mbed test -m TARGET_NAME -n tests-mbed_drivers-flashiap`.
 - Flash HAL: `mbed test -m TARGET_NAME -n tests-mbed_hal-flash`.
+
+## Troubleshooting
+
+1. For targets with VTOR, a target might have VTOR address defined to a hardcoded address as mentioned in the ``Linker script updates`` section.
```

Comment:
Should we link to the Linker script updates section instead of putting it in code? Also, please change "might have VTOR address" to "might have a VTOR address".